src/layouts/Default.astro

```diff
@@ -16,10 +16,10 @@ let { metaTitle } = Astro.props;
   rel="stylesheet"
   href={Astro.resolve("../styles/global.css")}
   >
-  <meta name="description" content="Dev Protocol lets GitHub users tokenize their OSS projects in a few clicks. Creating an OSS token allows projects to access sustainable funding, create underlying economic models for their project, and distribute incentives at scale. For the first time, OSS projects have a scalable solution to unlock their value.">
+  <meta name="description" content="Dev Protocol lets GitHub users tokenize their OSS projects in a few clicks. Creating an OSS token allows projects to access sustainable funding, create underlying economic models for their project, and distribute incentives at scale. For the first time, OSS projects have a scalable solution to unlock their value." />
```

Comment:
Does this make a difference after HTML5?

Reply:
> Does this make a difference after HTML5?

Not sure, but currently that's not working. I thought that would fix it. 🤷‍♂️

Reply:
It didn't fix it :(